spec/support/bulk_upload/sales_log_to_csv.rb

```diff
@@ -357,7 +413,7 @@ def to_2024_row
       log.proplen,
       log.value,
       log.grant,
-      log.discount,
+      log.discount || 0,
```

Comment:
I think there may be a bug around this being required when we might expect grant instead, but I'll raise that separately and leave this here for now.